Aliucord/src/main/java/com/aliucord/settings/Crashes.java

```diff
@@ -64,14 +69,64 @@ public Context getContext() {
     @SuppressLint("SetTextI18n")
     public void onViewBound(View view) {
         super.onViewBound(view);
-
         setActionBarTitle("Crash Logs");
 
         Context context = requireContext();
-        LinearLayout layout = (LinearLayout) ((NestedScrollView) ((CoordinatorLayout) view).getChildAt(1)).getChildAt(0);
+        int padding = Utils.getDefaultPadding();
+        int p = padding / 2;
+
+        File dir = new File(Constants.CRASHLOGS_PATH);
+        if (!dir.exists()) {
+            boolean res = dir.mkdir();
+            if (!res) logger.error("Failed to create crashlogs directory!", null);
+        }
+        File[] files = dir.listFiles();
+        assert files != null;
```

Comment:
```suggestion
        if (files == null) files = new File[0];
```

the assert will cause an unrecoverable crash if creating the dir failed